README.md

```diff
@@ -46,6 +46,68 @@ To format all the scala files in the project correctly:
 To check if there are any scalastyle errors, warnings or infos:
 > `sbt scalastyle`
 
+
+### Accessibility Test
+1. Keep this registration frontend service up and running. 
+
+
+2. Run the _"dockerized"_ accessibility assessment service:
+    ```sh
+    $ docker container run \
+          --name "accessibility-assessment" \
+          --detach --restart unless-stopped \
+          --publish 6010:6010 \
+          --env TARGET_IP='host.docker.internal' \
+          artefacts.tax.service.gov.uk/accessibility-assessment:latest
+    ```
+   Be aware that's a _"stateful"_ service. You may want to read more about it. See https://github.com/hmrc/accessibility-assessment
+   > NOTE  
+   > You may need an Intel x86_64 architecture as the Docker image for ARM may not be available yet. 
+
+
+3. Change your working directory to the acceptance test project and run the test with the right arguments:
+   ```sh
+   $ cd $WORKSPACE/economic-crime-levy-acceptance-tests
+   $ sbt \
+        -Dbrowser="chrome" \
+        -Denvironment="local" \
+        -Daccessibility.test="true"  \
+        "testOnly uk.gov.hmrc.test.ui.cucumber.runner.RunRegistration"
+   ```
+   > NOTE  
+   > You may need to install Google Chrome and the WebDriver for Chrome (see https://chromedriver.chromium.org/downloads)
+
+
+4. Once the above accessibility test completes, request the assessment service to assess (collect and assemble) the report pages:
+   ```sh
+   $ curl --verbose \
+          --request POST \
+          "http://localhost:6010/api/assess-pages"
+   ```
+   After a short while, query for the status:
+   ```sh
+   $ curl --verbose \
+          --request GET \
+          "http://localhost:6010/api/status"
+   ```
+   Keep querying until the status turns into `REPORT_READY`
+
+
+5. Extract the HTML report from the assessment service container
+   ```sh
+   $ docker container cp \
+       accessibility-assessment:/home/seluser/output/accessibility-assessment-report.html \
+      ./target
+   ```
+
+6. (Optional) Reset the accessibility service to its initial state (by removing the collected pages and any previously generated reports) in case you want to repeat the accessibility test again:
+  ```sh
+  $ curl --verbose \
+         --request POST \
+         "http://localhost:6010/api/app/reset"
+  ```
+
+
```

Comment:
This is great - but can we move these instructions to the readme within https://github.com/hmrc/economic-crime-levy-acceptance-tests please? this is where the automated UI tests are and also it makes sense for the other journeys there as well